### ISSUE TITLE
[FEATURE]: Embed JitsiMeet Flutter SDK within a widget to enable custom layouts?

### ISSUE THREAD

**Issue by zeckem19** [jitsi/jitsi-meet-flutter-sdk#24]
Currently when joininig or creating a room, the fullscreen view is launched.  is it possible to allow for some flexibility here so that i can have jitsi meet not take up the full screen?

**Comment by E2-Harish** [comment]
> Hey there! We are considering adding support for that, but are gauging interest first. What kind of layout do you have in mind?

It would be better if we can embed the view inside a Flutter widget such as Container

**Comment by zeckem19** [comment]
> > Hey there! We are considering adding support for that, but are gauging interest first. What kind of layout do you have in mind?
> 
> It would be better if we can embed the view inside a Flutter widget such as Container

yes @saghul allowing us to embed it within a Container would allow us flexibly size/ position it. currently i have to workaround with the iframe API instead

**Comment by saghul** [comment]
> > Hey there! We are considering adding support for that, but are gauging interest first. What kind of layout do you have in mind?
> 
> 
> 
> It would be better if we can embed the view inside a Flutter widget such as Container

Why?

**Comment by saghul** [comment]
> > > Hey there! We are considering adding support for that, but are gauging interest first. What kind of layout do you have in mind?
> 
> > 
> 
> > It would be better if we can embed the view inside a Flutter widget such as Container
> 
> 
> 
> yes @saghul allowing us to embed it within a Container would allow us flexibly size/ position it. currently i have to workaround with the iframe API instead

Why do you need to do that? What else are you showing besides the meeting?

Just trying to understand what the use case is.

**Comment by zeckem19** [comment]
> > > > Hey there! We are considering adding support for that, but are gauging interest first. What kind of layout do you have in mind?
> > 
> > 
> > > 
> > 
> > 
> > > It would be better if we can embed the view inside a Flutter widget such as Container
> > 
> > 
> > yes @saghul allowing us to embed it within a Container would allow us flexibly size/ position it. currently i have to workaround with the iframe API instead
> 
> Why do you need to do that? What else are you showing besides the meeting?
> 
> Just trying to understand what the use case is.

working on an app for users to take pictures, select relevant ones, before calling someone to discuss it. would like to show a gallery on half the screen, and jitsi meet on the other half.

**Comment by saghul** [comment]
Thanks for the feedback! We can't promise anything, but knowing this is helpful to see if / when to spend some time on this.

**Comment by E2-Harish** [comment]
> > > > > Hey there! We are considering adding support for that, but are gauging interest first. What kind of layout do you have in mind?
> > > 
> > > 
> > > > 
> > > 
> > > 
> > > > It would be better if we can embed the view inside a Flutter widget such as Container
> > > 
> > > 
> > > yes @saghul allowing us to embed it within a Container would allow us flexibly size/ position it. currently i have to workaround with the iframe API instead
> > 
> > 
> > Why do you need to do that? What else are you showing besides the meeting?
> > Just trying to understand what the use case is.
> 
> working on an app for users to take pictures, select relevant ones, before calling someone to discuss it. would like to show a gallery on half the screen, and jitsi meet on the other half.

I also have a similar usecase.
